nextflow.config

```diff
@@ -125,6 +122,13 @@ params {
     max_memory                 = '128.GB'
     max_cpus                   = 16
     max_time                   = '240.h'
+
+    // Schema validation default options
+    validationFailUnrecognisedParams = false
+    validationLenientMode            = false
+    validationSchemaIgnoreParams     = 'genomes,snpeff_version,vep_version,cf_ploidy'
+    validationShowHiddenParams       = false
+    validate_params                  = true
```

Comment:
it's not `validateParams`?
I don't follow this new pattern for the params :rofl:

Reply:
`validate_params` is an nf-core param, not an nf-validation/nextflow one